src/FaceIT.Hangfire.Tags.SqlServer/SqlTagsTransaction.cs

```diff
@@ -61,7 +61,7 @@ public SqlTagsTransaction(SqlServerStorageOptions options, IWriteOnlyTransaction
             if (_addCommand == null)
             {
                 _addCommand = transaction.GetType().GetTypeInfo().GetMethod(nameof(AddCommand),
-                    BindingFlags.NonPublic | BindingFlags.Instance);
+                    BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.Static);
```

Comment:
Add this to work with `Hangfire.SqlServer` > 1.8.6

https://github.com/HangfireIO/Hangfire/commit/c0d6290153abca71e08834f3133222e0c110063e